smart-contracts/contracts/mixins/MixinPurchase.sol

```diff
@@ -22,53 +22,22 @@ contract MixinPurchase is
   using SafeMath for uint;
 
   /**
-  * @dev Purchase function, public version, with no referrer.
-  * @param _recipient address of the recipient of the purchased key
-  */
-  function purchaseFor(
-    address _recipient
-  )
-    external
-    payable
-    onlyIfAlive
-  {
-    return _purchaseFor(_recipient, address(0));
-  }
-
-  /**
-  * @dev Purchase function, public version, with referrer.
+  * @dev Purchase function
   * @param _recipient address of the recipient of the purchased key
   * @param _referrer address of the user making the referral
-  */
-  function purchaseForFrom(
-    address _recipient,
-    address _referrer
-  )
-    external
-    payable
-    onlyIfAlive
-    hasValidKey(_referrer)
-  {
-    return _purchaseFor(_recipient, _referrer);
-  }
-
-  /**
-  * @dev Purchase function: this lets a user purchase a key from the lock for another user
-  * @param _recipient address of the recipient of the purchased key
   * This will fail if
   *  - the keyReleaseMechanism is private
```

Comment:
We could probably remove references to the "keyReleaseMechanism" as well( if we're not going to be introducing private or restricted locks). These are old and confusing to someone who doesn't know what they are.

Reply:
Hmm yea.  Actually none of those points are valid.  Deleted.